libs/core/src/components/pds-text/pds-text.tsx

```diff
@@ -90,7 +90,7 @@ export class PdsText {
   /**
    * If set or `true`, the text will be truncated. Must add a `width` to the element.
    */
-  @Prop() truncate?: boolean;
+  @Prop({ reflect: true }) truncate?: boolean;
```

Comment:
Noticed that truncation was not being applied correctly during the development of the small variant of the alert component. This corrects the issue.